### PR TITLE
test Julia 1.3, FreeBSD, add badge

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  image: freebsd-12-0-release-amd64
+task:
+  name: FreeBSD
+  env:
+    matrix:
+      - JULIA_VERSION: 1.0
+      - JULIA_VERSION: 1.1
+      - JULIA_VERSION: 1.2
+      - JULIA_VERSION: 1.3
+      - JULIA_VERSION: nightly
```
Comment (abelsiqueira) [inline .cirrus.yml]: Is there a `allow_failures` for cirrus?

Comment (dpo) [reply .cirrus.yml]: There is, but the syntax isn't very clear. Nightly works for other packages though, which is confusing.

Comment (abelsiqueira) [reply .cirrus.yml]: The error is not very descriptive.

**Comment by dpo** [comment]
Now it's working. I suspect it was because we didn't have a Project.toml.

**Pull request merged**